### PR TITLE
Static asset management

### PR DESCRIPTION
**Pull request by jimmcgaw**
In this PR: introducing bower for dependency management of static assets; update React components that use PropTypes to use new standalone proptypes package; encapsulate build process (sass compilation, react browserifying) into npm script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +30,20 @@ There is currently one public instance running at
...
-# Dev: watchify -t [ babelify --presets [ react es2015 ] ] src/index.js -o build/app.js
-browserify -t [ babelify --presets [ react es2015 ] ] src/index.js -o build/app.js
-go build
-./leapchat
-```
+To install and build static assets:
+
+``` $ npm install ```
+
+``` $ npm run build ```
```
Comment (elimisteve) [inline README.md]: @jimmcgaw I made these tweaks, and now everything works great for me: https://github.com/cryptag/leapchat/pull/41/commits/921808cf455b664f550463019fc735bc390e675d

Comment (elimisteve) [inline README.md]: Rebased my tweaks: https://github.com/cryptag/leapchat/pull/41/commits/adfe9f8a9d596501e0962dd0449f017fdc9e183b

Comment (elimisteve) [inline README.md]: :+1: LGTM. If that looks good to you, let's merge baby merge!

Comment (elimisteve) [inline README.md]: ...force-pushed: 2d3159e8de066bd1805f5f5cdc74f8234c8a7aa6

File: README.md [hunk 2]
```diff
@@ -30,18 +30,31 @@ There is currently one public instance running at
...
+
+## Install and Run
+
 To install and build static assets:
 
 ``` $ npm install ```
 
+``` $ mkdir build ```
+
+``` $ cd static/ && bower install && cd .. ```
```
Comment (elimisteve) [inline README.md]: @jimmcgaw Without this I wasn't getting jQuery and Bootstrap, and running `gulp inject` directly was giving me problems.  How's this for a solution?

Comment (jimmcgaw) [inline README.md]: Good catch. Eventually I'd like to move everything into npm tasks (that refer to node_modules/.bin/gulp, etc, so global binaries aren't needed.

**Pull request merged**